src/box/space.c

```diff
@@ -325,6 +326,123 @@ space_unpin_defaults(struct space *space)
 	}
 }
 
+/**
+ * Set all the events associated with space: on_replace and before_replace,
+ * bound by name and by id. If argument on_recovery is true, recovery events
+ * are set instead of regular ones.
+ */
+static void
+space_set_events(struct space *space, bool on_recovery)
+{
+	assert(space->def != NULL);
+	/*
+	 * Space events - on_replace and before_replace events. All event names
+	 * must be declared in the same order.
+	 */
+	struct space_event *space_events[] = {
+		&space->on_replace_event,
+		&space->before_replace_event,
+	};
+	/* Format strings for names of events associated with space by id. */
+	static const char * const regular_event_by_id_fmt[] = {
```

Comment:
I would not use 'static' here.

Reply:
Cannot use just `const char *` because of checkpatch.